content/user/upgrading/db_only_upgrade.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Database-Only Upgrade Instructions
+description: Upgrading if you don't want to keep your customizations 
+category: Upgrading
+weight: 10
+---
+
+Doing a database only upgrade might be appropriate if you are coming from Zen Cart 1.5.4 or older version, and you don't have time or budget for a proper upgrade.  You will lose your customizations, but you won't lose your data and you'll be using the latest software, which is more secure. 
```

Comment:
Why 1.5.4 ?

Reply:
It's the last release before responsive classic.  But I'm open to other ideas.